### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by Rany615**
regex brackets removed, from peoples names in city,
data source not primary update from aka,
no update when theres no akaRecord,
fix comparison of personCopy and person,
fix dates birthdate and dischargeDay,
fix city_name to use fn.dataSources

### CODE REVIEW

**Review by NoamYes — COMMENTED** [review]
Please Review

File: src/util/diffsHandlerUtils/addedDataHandler.js [hunk 1]
```diff
@@ -145,20 +146,22 @@ module.exports = async ({ addedData, dataSource }, extraData) => {
...
                 }
                 DataModel.updateDeepDiff = diff(
-                    [DataModel.person],
                     [personCopy],
+                    [DataModel.person_ready_for_kartoffel],
                     KeyForComparison,
                     { updatedValues: 4 }
                 ).updated[0];
 
+                try{
```
Comment (NoamYes) [inline src/util/diffsHandlerUtils/addedDataHandler.js:155]: whats the purpose of this try catch?

**Review by NoamYes — COMMENTED** [review]
Seems fine

File: src/util/diffsHandlerUtils/addedDataHandler.js (hunk 1 above)
Comment (Rany615) [inline src/util/diffsHandlerUtils/addedDataHandler.js:155]: to remove

**Review by NoamYes — APPROVED** [review]

**Pull request merged**